recipe/meta.yaml

```diff
@@ -57,21 +57,23 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
         - libpng
-        - openjpeg
+        - openjpeg  # [not py35]
```

Comment:
@pelson we need to pin version in `openjpeg`. Let me know if I should do that manually or if you want to send a MTN PR.